src/neo/SmartContract/Native/NeoToken.cs

```diff
@@ -348,6 +348,18 @@ public ECPoint[] GetCommittee(DataCache snapshot)
             return GetCommitteeFromCache(snapshot).Select(p => p.PublicKey).OrderBy(p => p).ToArray();
         }
 
+        /// <summary>
+        /// Get account state.
+        /// </summary>
+        /// <param name="snapshot">The snapshot used to read data.</param>
+        /// <param name="account">account</param>
+        /// <returns>The amount of the fees.</returns>
+        [ContractMethod(CpuFee = 1 << 15, RequiredCallFlags = CallFlags.ReadStates)]
+        public NeoAccountState GetAccountState(DataCache snapshot, UInt160 account)
+        {
+            return snapshot.GetAndChange(CreateStorageKey(Prefix_Account).Add(account))?.GetInteroperable<NeoAccountState>();
```

Comment:
You should use `TryGet` instead of `GetAndChange`.

Reply:
Done